CHANGELOG-windows.md

```diff
@@ -22,7 +24,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Service: `HandEntered` & `HandExited` events, sent when the active hand enters and exits the interaction zone (if enabled) respectively
```

Comment:
🤷 Do we want to track this change? Is it user visible/impactful? Conscious that if we don't a lot of service changes will be undocumented though...

Reply:
Not really sure. I don't mind removing it if it's not helpful, but I don't think there's any harm in it being there either 🤷